### ISSUE TITLE
argument error using scheduler

### ISSUE THREAD

**Issue by magistermilitum** [mittagessen/kraken#483]
Using the last version (4.3.5) an error raises when using an scheduler other that constant in the trainer, v.g: ketos train - -g 0.5 --schedule step -ss 20

Error says:
lr_scheduler_step() missing 1 required positional argument: 'metric'

**Comment by mittagessen** [comment]
Thanks. I had missed some of the changes that lightning 2.x introduced in the callback signatures. Unfortunately, they didn't document them....
